model/object.go

```diff
@@ -31,12 +31,12 @@ import (
 //     parsed to int32
 //   - raw		- holds any not typed value, replaces the interface{} behavior.
 //
-// +kubebuilder:validation:Type=object
+// +kubebuilder:validation:Type=string
```

Comment:
Isn't this annotation in this position validating the entire Object struct, forcing it to be a string?